.github/labeler.yml

```diff
@@ -0,0 +1,5 @@
+"[Status] Needs Package Release":
+- 'packages/*'
+- 'packages/*/*'
+- 'packages/*/*/*'
+- 'packages/*/*/*/*'
```

Comment:
Does `/**/*` not work to cover subdirs?

Pardon my ignorance if it doesn't, I'm not familiar with tech used here. ;-)

Reply:
This was the format recommended in the docs:
https://help.github.com/en/articles/workflow-syntax-for-github-actions#example-matching-paths-in-nested-directories